ostelco-ios-client/Coordinators/OnboardingCoordinator.swift

```diff
@@ -49,23 +49,20 @@ class OnboardingCoordinator {
         primeAPI.loadContext()
             .done { (context) in
                 self.localContext.serverIsUnreachable = false
-                if let region = context.toLegacyModel().getRegion()?.region {
-                    self.localContext.selectedRegion = Region(gqlRegion: region)
-                }
                 
                 UserManager.shared.customer = context.customer
                 let stage = self.stageDecider.compute(context: context.toLegacyModel(), localContext: self.localContext)
                 self.afterDismissing {
                     self.navigateTo(stage)
                 }
-            }.recover { error in
-                self.localContext.serverIsUnreachable = (error as NSError).code == -1004
-                let context: Context? = nil
-                let stage = self.stageDecider.compute(context: context, localContext: self.localContext)
-                self.afterDismissing {
-                    self.navigateTo(stage)
-                }
+        }.recover { error in
+            self.localContext.serverIsUnreachable = (error as NSError).code == -1004
```

Comment:
-1004 is a magic number.  What does it mean? Either add a comment, or declare a constant with an explanatory name.

Reply:
-1004 means the server is unreachable. That's why we change `serverIsUnreachable` to true when the code equals that number.